analysers/analyser_merge_bicycle_parking_FR_paris.py

```diff
@@ -48,8 +48,10 @@ def __init__(self, config, logger = None):
         self.init(
             u"https://opendata.paris.fr/explore/dataset/stationnement-voie-publique-emplacements/information/",
             u"Stationnement sur voie publique - emplacements vélos",
-            CSV(Source(attribution = u"Ville de Paris", millesime = "02/2018",
-                    fileUrl = u"https://opendata.paris.fr/explore/dataset/stationnement-voie-publique-emplacements/download/?format=csv&timezone=Europe/Berlin&use_labels_for_header=true"),
+            CSV(SourceOpenDataSoft(attribution = u"Ville de Paris",
+                    base_url = "https://opendata.paris.fr",
+                    dataset = "stationnement-voie-publique-emplacements",
+                    timezone = "Europe/Berlin"),
```

Comment:
Ca we omit the timezone of have GMT default?

Reply:
Yeah I had the same reaction seeing this.

For the first pass, to raise acceptance of this PR I made the least number of changes.

I was planning on stacking breaking or controversial changes on top of accepted changes, such as this hardcoded timezone, switch to ISO-8601 millesime source date, dynamic attribution etc.

Reply:
I removed hardcoded timezones as they are not used in the analyses, and switched to ISO-8601 dates. I'll maybe fix some attributions in another PR.